README.md

```diff
@@ -16,27 +14,26 @@ Require this package, with [Composer](https://getcomposer.org/), in the root dir
 composer require wordplate/translator
 ```
 
-Login to the WordPress administrator dashboard and active the [Polylang](https://wordpress.org/plugins/polylang/) plugin.
+Login to the WordPress administrator dashboard and activate the [Polylang](https://wordpress.org/plugins/polylang/) plugin. Go to the Polylang settings page and add at least one language.
 
 ## Usage
 
-Register the translations with the `register_translations` method.
+Register the translations with the `register_translations` method. They will now be available for translation on the 'Strings translations' page in the WordPress administrator dashboard.
 
 ```php
 register_translations([
     'general' => [
-        'general-language-english' => 'The english language',
-        'general-language-swedish' => 'The swedish language',
+        'English' => 'The english language',
```

Comment:
As far as I can understand pll_translate_string() works similar to __(). That is, the given parameter should be seen as the actual string and not some sort of lookup key. That way trans()/pll_translate_string() works straight away in the templates (and isn't dependent on someone having to supply a string in wp-admin first). This means that pll_register_string() and register_translations() should receive strings instead of keys.